test_teardown.stress_ng.yml

```diff
@@ -0,0 +1,18 @@
+---
+# Removal of stress-ng tool and all it's dependencies (checked via yum.log),
+# mainly to make sure we don't keep other tools from EPEL installed.
+- hosts: usm_nodes
+  remote_user: root
+  tasks:
+    - name: Remove stress-ng tool and direct dependencies
+      yum:
+        name: "{{ item }}"
+        state: absent
+      with_items:
+        - stress-ng
+        - libbsd
+        - lksctp-tools
```

Comment:
This `lksctp-tools` is from rhel channel, @dahorak do you think we want to remove this here as well?

Reply:
If we include `lksctp-tools`, we should also consider `libaio`, as this was also installed as dependency of `stress-ng` (from rhel repo) - at least on my setup. But I wouldn't remove them at all, because we are not easily able to distinguish, if they were installed only as dependency of `stress-ng` or as a dependency also for something else and it might remove unintentionally some other packages.

Reply:
Makes sense, I'm going to remove `lksctp-tools` from there.